src/pages/PublishedSchedule/PublishedSchedule.jsx

```diff
@@ -52,60 +62,146 @@ const PublishedSchedule = () => {
     setAllSeasons(data);
   };
 
+  const renderTable = async () => {
+    if (selectedSeason.length) {
+      const [seasonType, seasonYear] = selectedSeason.split(' ');
+      const { data } = await NPOBackend.get(
+        `/published-schedule/season?season=${seasonType}&year=${seasonYear}`,
+      );
+      setAllEvents(data);
+      if (!searchedDate.length) {
+        setEventData(data);
+      }
+    }
+  };
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The 'renderTable' function makes the dependencies of useEffect Hook (at line 96) change on every render. To fix this, wrap the definition of 'renderTable' in its own useCallback() Hook.